src/commands/lambda/flare.ts

```diff
@@ -229,6 +242,62 @@ export class LambdaFlareCommand extends Command {
   }
 }
 
+/**
+ * Obfuscate the environment variables in a Lambda function configuration
+ * @param config
+ */
+export const obfuscateConfig = (config: FunctionConfiguration) => {
```

Comment:
I think the correct name for this would be `maskConfig`, obfuscation is the process, but what you are doing here is a tactic of it, you are masking the data.